app/domain/authentication/authn_jwt/identity_providers/create_identity_provider.rb

```diff
@@ -22,16 +22,22 @@ def call
         private
 
         def create_identity_provider
-          identity_provider = @identity_from_decoded_token_class.new(@authentication_parameters)
-          if identity_provider.identity_available?
+          identity_from_decoded_token_provider = @identity_from_decoded_token_class.new(@authentication_parameters)
+          identity_from_url_provider = @identity_from_url_provider_class.new(@authentication_parameters)
+
+          if identity_from_decoded_token_provider.identity_available and identity_from_url_provider.identity_available
```

Comment:
what about they both unavailable ?

Reply:
We throw error

Reply:
we want to throw the same error as they both configured